examples/dnn_yolo_train_ex.cpp

```diff
@@ -255,8 +255,7 @@ try
     // When computing the objectness loss in YOLO, predictions that do not have an IoU
     // with any ground truth box of at least options.iou_ignore_threshold, will be
     // treated as not capable of detecting an object, an therefore incur loss.
-    // Predictions above this threshold will be ignored, i.e. will not contribute to the
-    // loss. Good values are 0.7 or 0.5.
+    // Similarly, predictions above this threshold are considered correct predictions by the loss. Typical settings for this threshold are in the range 0.5 to 0.7.
```

Comment:
Add a new line here so this line isn't so long.

Reply:
Done!